lib/ret/login.ex

```diff
@@ -22,4 +24,33 @@ defmodule Ret.Login do
     |> unique_constraint(:identifier_hash)
     |> put_assoc(:account, account)
   end
+
+  def update_identifier_hash(%{old_email: old_email, new_email: new_email} = _) do
+    old_identifier_hash = Account.identifier_hash_for_email(old_email)
+    new_identifier_hash = Account.identifier_hash_for_email(new_email)
+    login = Repo.get_by(Login, identifier_hash: old_identifier_hash)
+    case update_identifier_hash_for_login(login, new_identifier_hash) do
+      {:error, :no_login} -> {:error, :no_account_for_old_email}
```

Comment:
Why not let `update_identifier_hash_for_login/2`, which is only used right here, return the right value?

Reply:
Resolved in eee56865456bf34ae00958478c1ed8e15a49007b